ruby/ql/lib/codeql/ruby/frameworks/ActionController.qll

```diff
@@ -118,6 +118,25 @@ class ParamsSource extends RemoteFlowSource::Range {
   override string getSourceType() { result = "ActionController::Metal#params" }
 }
 
+/**
+ * A call to the `cookies` method to fetch the request cookies.
+ */
+abstract class CookiesCall extends MethodCall {
+  CookiesCall() { this.getMethodName() = "cookies" }
+}
+
+/**
+ * A `RemoteFlowSource::Range` to represent accessing the
+ * ActionController cookies available via the `cookies` method.
+ */
+class CookiesSource extends RemoteFlowSource::Range {
+  CookiesCall call;
+
+  CookiesSource() { this.asExpr().getExpr() = call }
+
+  override string getSourceType() { result = "ActionController::Metal#params" }
```

Comment:
s/params/cookies

But you can get rid of this change completely, if we merge https://github.com/github/codeql/pull/7341

Reply:
Thanks, this was a WIP change that I included by accident. I'll take a look at #7341 today.